website/i18n/en.json

```diff
@@ -0,0 +1,161 @@
+{
+  "_comment": "This file is auto-generated by write-translations.js",
```

Comment:
If it really is, shouldn't this file be mentioned in .gitignore?

Reply:
I have added this to .gitignore.